unlock-app/src/actions/lock.js

```diff
@@ -21,6 +22,12 @@ export const updateLock = (address, update) => ({
   update,
 })
 
+export const updateLockPrice = (lock, price) => ({
```

Comment:
This is good but please add a test to `actions/lock.test.js` to make sure the right action object is created.